socorro/processor/mozilla_transform_rules.py

```diff
@@ -628,53 +633,35 @@ def _get_version_data(self, product, version, build_id):
         is 54.0). This database call returns the actual version number of said
         build (i.e. 54.0b3 for the previous example).
         """
```

Comment:
We should document arguments, returns, and also that this can raise `requests.RequestException`.